src/main/java/org/graylog/plugins/outputs/neo4j/Neo4jOutput.java

```diff
@@ -151,9 +104,18 @@ public void stop() {
         public ConfigurationRequest getRequestedConfiguration() {
             final ConfigurationRequest configurationRequest = new ConfigurationRequest();
 
+            final Map<String, String> protocols = ImmutableMap.of(
+                    "HTTP", "HTTP",
+                    "Bolt", "Bolt");
+
+            configurationRequest.addField(new DropdownField(
+                            CK_PROTOCOL, "Neo4J Protocol", "HTTP", protocols,
+                            "The protocol used to connect to Neo4J",
+                            ConfigurationField.Optional.NOT_OPTIONAL));
+
             configurationRequest.addField(new TextField(
                             CK_NEO4J_URL, "Neo4j URL", "http://localhost:7474/db/data",
-                            "URL to Neo4j",
+                            "URL to Neo4j, default for Bolt: bolt://localhost:7687/",
```

Comment:
Help is displayed after the field, similar to the instructions for the cypher query. Isn't that the right place?
![neo4j url field](https://cloud.githubusercontent.com/assets/6745661/17809003/88054c68-660b-11e6-9a8f-d7a0110ae9f0.png)